polar-core/src/resource_block.rs

```diff
@@ -266,7 +266,7 @@ pub struct ResourceBlock {
 #[derive(Clone, Default)]
 pub struct ResourceBlocks {
     /// Map from resource (`Symbol`) to the declarations in that resource's block.
-    declarations: HashMap<Term, Declarations>,
+    pub declarations: HashMap<Term, Declarations>,
```

Comment:
We might not've followed this advice previously given the other `pub` fields here, but in general if you want to expose read-only access to a field a good strategy is to make a getter method that returns a reference to the field instead of making the field `pub`.

Reply:
👍🏻 refactored to use a getter.